src/generator/templates/api/lib/mbuild.mako

```diff
@@ -134,7 +134,7 @@ impl${mb_tparams} ${CALL_BUILDER_MARKERT_TRAIT} for ${ThisType} {}
 
 impl${mb_tparams} ${ThisType}
 where
-    S: tower_service::Service<Uri> + Clone + Send + Sync + 'static,
+    S: tower_service::Service<http::Uri> + Clone + Send + Sync + 'static,
```

Comment:
Google Wallet API has a `Uri` data type, so it would be worth avoiding conflict of names with `http::Uri`.
https://developers.google.com/wallet/generic/rest/v1/Uri?hl=en

Reply:
I thought so, thanks for the fix.